assets/js/modules/idea-hub/datastore/__fixtures__/draft-post-ideas.json

```diff
@@ -0,0 +1,52 @@
+[
```

Comment:
Let's update `draft-post-ideas.json` and `published-post-ideas.json` files to use the correct format for its items which must include `postID`, `postEditURL`, and `postURL` (for published posts only) fields.

Reply:
Updated @eugene-manuilov